src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogQueuePublisher.java

```diff
@@ -85,10 +88,16 @@ protected void doRun() throws Exception {
                 
                 String job_name;
                 Task task = item.task;
-                if (task instanceof FreeStyleProject){ 
+                if ((task instanceof FreeStyleProject) | (task instanceof WorkflowJob)) { 
```

Comment:
Can the task really be of type WorkflowJob here ?

Reply:
`WorkflowJob` implements `Queue.Task`, so I wanted to be safe https://javadoc.jenkins.io/plugin/workflow-job/org/jenkinsci/plugins/workflow/job/WorkflowJob.html

Reply:
Have you seen that condition being true?
I'm afraid of the following:
- `getFullDisplayName` on WorkflowJob could return a build ID and cause context explosion.
- task could be an instance of both WorkflowJob and ExecutorStepExecution.PlaceholderTask leading to the wrong code being run

Reply:
I've seen it true in the tests but not in a jenkins instance. I can either get rid of this check or move it after the `ExecutorStepExecution.PlaceholderTask` check.

Reply:
> I can either get rid of this check

Seems safer for now until we find a real-life use case for something like this